src/api/handlers/channel.ts

```diff
@@ -240,10 +240,11 @@ export async function sendMessage(
           replied_user: content.mentions.repliedUser,
         }
         : undefined,
-      ...(content.replyMessageID
+      ...(content.replyMessageID || content.failReplyIfNotExists
```

Comment:
Discords default is true
```suggestion
      ...(content.replyMessageID || content.failReplyIfNotExists === false
```